package.json

```diff
@@ -9,34 +9,26 @@
     "test:unit": "vitest --environment jsdom --root src/"
   },
   "dependencies": {
-    "@headlessui/vue": "^1.7.4",
-    "@heroicons/vue": "^2.0.13",
-    "@ownego/polaris-vue": "^1.3.0",
-    "@tailwindcss/line-clamp": "^0.4.2",
-    "@tailwindcss/typography": "^0.5.8",
     "core-js": "^3.8.3",
     "pinia": "^2.0.26",
-    "postcss-cli": "^10.1.0",
-    "postcss-nested": "^6.0.0",
-    "postcss-nesting": "^10.2.0",
-    "remixicon": "^2.5.0",
-    "sass": "^1.56.1",
-    "typeface-inter": "^3.18.1",
     "vue": "^3.2.45",
     "vue-router": "^4.1.6"
   },
   "devDependencies": {
+    "@headlessui/vue": "^1.7.5",
     "@tailwindcss/forms": "^0.5.3",
+    "@tailwindcss/line-clamp": "^0.4.2",
+    "@tailwindcss/typography": "^0.5.8",
     "@vitejs/plugin-vue": "^3.2.0",
-    "@vue/eslint-config-prettier": "^7.0.0",
     "@vue/test-utils": "^2.2.4",
     "autoprefixer": "^10.4.13",
-    "eslint": "^8.22.0",
-    "eslint-plugin-vue": "^9.3.0",
     "jsdom": "^20.0.3",
     "postcss": "^8.4.19",
-    "prettier": "^2.7.1",
```

Comment:
why are these deleted?

Reply:
These are not required for Frontend and UI development. If your development team require the then you can install.

Reply:
What are the conflicts/issues you have in merging... Can you share?

Reply:
We are here to develop frontend and UI. So please expect us to UI and fronted related issues fixes.
Now, we should start 2nd milestone.